samples/node-http-server.xml

```diff
@@ -0,0 +1,51 @@
+<!--
+  MIT License
+
+  Copyright (c) 2008-2020 Kohsuke Kawaguchi, Sun Microsystems, Inc., CloudBees,
```

Comment:
If you want to keep the license header, please update it so that it refers you or just WinSW Contributors + keep the right date. Also, IMHO the license header can be just deleted so that visitors see the actual content

Reply:
I have copied from other samples, but I agree with you! it's better delete it